src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -202,6 +187,8 @@ class SidebarPanels extends React.Component<Props> {
                         )}
                     />
                 )}
+                {/* This is before the activity feed subroutes because otherwise it doesn't get
+                    matched properly due to the :sidebar wildcard. */}
                 {hasVersions && (
                     <Route
                         path="/:sidebar/versions/:versionId?"
```

Comment:
Can we change this to `path="/:sidebar(activity|details)/versions/:versionId?"` to clarify where it's meant to be used?